documentation/modules/exploit/multi/http/gitlab_exif_rce.rb

```diff
@@ -0,0 +1,182 @@
+## Vulnerable Application
```

Comment:
Our linter should've detected this, but looks like this is a `.rb` file instead of `.md` 😄